public/plugin.tsx

```diff
@@ -138,11 +161,35 @@ export class SecurityPlugin implements Plugin<SecurityPluginSetup, SecurityPlugi
       })
     );
 
+    if (config.multitenancy.enable_aggregation_view) {
+      deps.savedObjectsManagement.columns.register(({
+        id: 'tenant_column',
+        euiColumn: {
+          field: 'namespaces',
+          name: <div>Tenant</div>,
+          dataType: 'string',
+          render: (value: any[][]) => {
+            let text = value[0][0];
```

Comment:
Is there documentation somewhere explaining why `namespaces` is a 2d array? 

Curious, what do you think of making this `value.flat()[0]`?

Reply:
Great idea! I've changed it now.